File: src/targets/python/requests/fixtures/query.py
import requests

<<<<<<< HEAD
url = "https://httpbin.org/anything?foo=bar&foo=baz&baz=abc&key=value"

response = requests.get(url)

print(response.text)
=======
url = "http://mockbin.com/har"

querystring = {"foo":["bar","baz"],"baz":"abc","key":"value"}

response = requests.get(url, params=querystring)

print(response.json())
>>>>>>> upstream/master

File: src/targets/r/httr/fixtures/query.r
library(httr)

<<<<<<< HEAD
url <- "https://httpbin.org/anything"

queryString <- list(
  foo = "bar,baz",
  baz = "abc"
=======
url <- "http://mockbin.com/har"

queryString <- list(
  perPage = "100",
  page = "1"
>>>>>>> upstream/master
)

response <- VERB("GET", url, query = queryString, content_type("application/octet-stream"))

content(response, "text")